src/example_behaviors/src/sam2_segmentation.cpp

```diff
@@ -0,0 +1,137 @@
+#include <spdlog/spdlog.h>
+
+#include <example_behaviors/sam2_segmentation.hpp>
+
```

Comment:
What are the iwyu rules here? Should you be
```suggestion
#include <sensor_msgs/msg/image.hpp>
```